metadata.yaml

```diff
@@ -1,16 +1,18 @@
 name: mattermost-k8s
 display-name: Mattermost
-summary: Mattermost is a flexible, open source messaging platform that enables secure team collaboration.
+summary: Messaging platform for teams.
```

Comment:
Is "Messaging platform for team collaboration" too long?

Reply:
I've expanded it to your proposal for now. This may be shortened again when I use operator-workflows and it's numerous checks.